### ISSUE TITLE
Secrets in redis.conf are exposed when exporting config metrics

### ISSUE THREAD

**Issue by mkoser** [oliver006/redis_exporter#609]
**Describe the problem**
Secret values like `masterauth` / `requirepass` are included in `redis_config_key_value` metrics when exporting config metrics.

**What version of redis_exporter are you running?**
v1.35.0

**Running the exporter**

```shell
/usr/local/bin/redis_exporter

## ENV
REDIS_ADDR="redis://localhost:6379"
REDIS_EXPORTER_EXPORT_CLIENT_LIST=true
REDIS_EXPORTER_INCL_CONFIG_METRICS=true
REDIS_EXPORTER_INCL_SYSTEM_METRICS=true
REDIS_EXPORTER_IS_CLUSTER=true
REDIS_EXPORTER_PING_ON_CONNECT=true
REDIS_EXPORTER_REDIS_ONLY_METRICS=true
REDIS_EXPORTER_WEB_LISTEN_ADDRESS=":9101"
REDIS_PASSWORD=******
```

**Expected behavior**
`masterauth` / `requirepass` values are excluded or obfuscated in the metric.

**Screenshots**

**Additional context**

**Comment by oliver006** [comment]
Oh fun, that's not good. I'll add something to suppress those metrics.
I'll definitely skip masterauth and requirepass, do you have a list of other metrics to exclude?

**Comment by oliver006** [comment]
Released as [v1.35.1](https://github.com/oliver006/redis_exporter/releases/tag/v1.35.1)
